gradio/templates.py

```diff
@@ -262,31 +288,60 @@ class PlayableVideo(components.Video):
 
     def __init__(
         self,
-        value: str | Callable | None = None,
+        value: str
+        | Path
+        | tuple[str | Path, str | Path | None]
+        | Callable
+        | None = None,
         *,
-        format: Literal["mp4"] | None = "mp4",
-        sources: list[Literal["upload", "webcam"]] = ["upload"],  # noqa: B006
+        format: str | None = "mp4",
+        sources: list[Literal["upload", "webcam"]] | None = None,
```

Comment:
switching to none because it's bad practice to have mutable default arguments in constructors